src/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -748,20 +752,88 @@ const UsersConfigurationRow = ({
     );
 };
 
+// This method needs to be outside of component as re-render would create a new instance of debounce
+const validateKey = debounce(500, (key, setKeyObject) => {
+    if (isEmpty(key))
+        return;
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1150-20231003-232224-a08f566d-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)